content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Windows containers in Kubernetes
+content_type: concept
+weight: 65
+---
+
+<!-- overview -->
+
+Windows एप्लिकेशन कई संगठनों में चलने वाली सेवाओं और अनुप्रयोगों के एक बड़े हिस्से का गठन करते हैं। [Windows कंटेनर](https://aka.ms/windowscontainers) प्रक्रियाओं और पैकेज निर्भरता को समाहित करने का एक तरीका प्रदान करते हैं, जिससे DevOps प्रथाओं का उपयोग करना और विंडोज अनुप्रयोगों के लिए क्लाउड नेटिव पैटर्न का पालन करना आसान हो जाता है। 
```

Comment:
> [Windows कंटेनर](https://aka.ms/windowscontainers) प्रक्रियाओं और पैकेज निर्भरता को समाहित करने का एक तरीका प्रदान करते हैं

Do we intend for this to be plural?

Reply:
Can you please clarify, plural for what?

Reply:
The word container